reactor-core/src/main/java/reactor/core/publisher/FluxSwitchOnFirst.java

```diff
@@ -902,43 +936,16 @@ public Object scanUnsafe(Attr key) {
 	}
 
 	static final class SwitchOnFirstConditionalControlSubscriber<T>
-			extends Operators.DeferredSubscription
-			implements InnerOperator<T, T>, ControlSubscriber<T>,
-			           Fuseable.ConditionalSubscriber<T> {
+			extends SwitchOnFirstControlSubscriber<T>
```

Comment:
nice, makes sense to have one extend the other and this gets rid of the interface 👍